pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -172,6 +172,24 @@ func (s *Service) getAPIServerLBSpec(elbName string, lbSpec *infrav1.AWSLoadBala
 		scheme = *lbSpec.Scheme
 	}
 
+	// The default API health check is TCP, allowing customization to HTTP or HTTPS when HealthCheckProtocol is set.
+	apiHealthCheckProtocol := infrav1.ELBProtocolTCP.String()
```

Comment:
You shouldn't need the `.String()` calls here and on L190, I don't think.

Reply:
Good point, thanks. Removed.